<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>280</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>小猪比机器人</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>376</width>
     <height>260</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_run">
    <attribute name="title">
     <string>运行配置</string>
    </attribute>
    <widget class="QGroupBox" name="runBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>140</height>
      </rect>
     </property>
     <property name="title">
      <string>运行配置</string>
     </property>
     <widget class="QLineEdit" name="portLine">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>30</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>端口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>30</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>主机</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hostLine">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>其他配置请前往机器人设置页面设置</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runBtn">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>启动服务器</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>运行</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stopBtn">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>停止RunThread线程</string>
      </property>
      <property name="text">
       <string>停止</string>
      </property>
     </widget>
    </widget>
    <widget class="QProgressBar" name="cpuBar">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QProgressBar" name="memBar">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>160</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>110</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU使用率</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>110</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>内存使用率</string>
     </property>
    </widget>
    <widget class="QLabel" name="cpuLab">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="memLab">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>190</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_log">
    <attribute name="title">
     <string>运行日志</string>
    </attribute>
    <widget class="QTextBrowser" name="logTextarea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>210</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_plugin">
    <attribute name="title">
     <string>插件管理</string>
    </attribute>
    <widget class="QListView" name="pluginsList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>351</width>
       <height>180</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="debugBtn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>启动服务器</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
    <widget class="QPushButton" name="howToDevBtn">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>启动服务器</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>如何开发</string>
     </property>
    </widget>
    <widget class="QPushButton" name="websiteBtn">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>停止RunThread线程</string>
     </property>
     <property name="text">
      <string>官网</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_config">
    <attribute name="title">
     <string>机器人设置</string>
    </attribute>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>373</width>
       <height>240</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>373</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>371</width>
        <height>238</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>371</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
